crates/bevy_render/src/view/mod.rs

```diff
@@ -876,6 +878,12 @@ impl ViewDepthTexture {
     }
 }
 
+/// A number that starts at 0 and increments by 1 every frame.
+///
+/// This is useful for determining which frame cached data belongs to.
+#[derive(Resource, Default, Deref, DerefMut)]
+pub struct FrameNumber(pub u32);
```

Comment:
The FrameCount resouce already provides that.

Reply:
I was just wondering whether we had this today.

Reply:
Oh cool, thanks. I removed `FrameNumber` in favor of `FrameCount`.